wordview/text_analysis/core.py

```diff
@@ -307,9 +368,7 @@ def get_pos(tagged_tokens: List[Tuple[str, str]], goal_pos: str) -> List:
     doc_lengths = []
     sentence_lengths = []
     token_to_count_dict: Dict[str, int] = {}
-    NNs: Dict[str, int] = {}
-    JJs: Dict[str, int] = {}
-    Vs: Dict[str, int] = {}
+    pos_counts: Dict[str, dict] = {pos: {} for pos in tags_list}
```

Comment:
The name is a little bit confusing. Can you please change it to reflect the content?
Especially when you compare it to the exact same variable name in wrapper which has different content and is derived from this same variable: 
`self.pos_counts = {k: len(v) for k, v in self.analysis.pos_counts.items()}`

Reply:
Thanks for pointing that out. The names were confusing indeed. I changed it to word_count_by_pos.